sdk/mobile-center-crashes/src/main/java/com/microsoft/azure/mobile/crashes/utils/ErrorLogHelper.java

```diff
@@ -44,6 +45,17 @@
     static final String ERROR_DIRECTORY = "error";
 
     /**
+     * For huge stack traces such as giant StackOverflowError, we keep only beginning and end of frames according to this limit.
+     */
+    @VisibleForTesting
```

Comment:
It doesn't do anything exception documentation. You can move this to Crashes, remove the annotation or leave as it is.